CDS links post (#6684)
GoogleChrome/web.dev: src/site/content/en/blog/cds2021-updates/index.md

---
layout: post
title: Everything announced at Chrome Dev Summit 2021
subhead: A roundup of all the key announcements from the 2021 Chrome Dev Summit, with the links you need to find out more.
hero: image/kheDArv5csY6rvQUJDbWRscckLr1/gK7MVviHglsjJNEDG6Au.png
alt: Chrome Dev Summit 2021
authors:
  - rachelandrew
description: >
  A roundup of all the key announcements from the 2021 Chrome Dev Summit, with the links you need to find out more.
date: 2021-11-03
tags:
  - blog
---

TO DO: add recap video

This post rounds up all the key announcements from the 2021 Chrome Dev Summit, with the links you need to find out more.

## Make the web more interoperable 

Web compatibility is one of the biggest challenges for web developers, 
and so Google has been working with other browser vendors to fix the top five compatibility pain points:  
[flexbox](/learn/css/flexbox/), 
[grid](/learn/css/grid/), 

> position: sticky, 

Reviewer: ```suggestion
`position: sticky`, 
```

> [aspect-ratio](/aspect-ratio/), 

Reviewer: ```suggestion
[`aspect-ratio`](/aspect-ratio/), 
```

> and [transforms](/learn/css/transforms/).

Reviewer: This link doesn't work, there's this though `learn/css/transitions/#transform`

Author: ```suggestion
and [transforms](/learn/css/transitions/#transform).
```

The work has already led to improvements in these key areas. 
You can read about those in the [Compat 2021 mid-year update](https://web.dev/compat2021-midyear/).

{% Aside %}
The Microsoft Edge team took on the work of rewriting CSS grid layout as part of the Compat 2021 effort. 
You can read about their work in the post 
[Compat2021: Improving CSS Grid compatibility with GridNG](https://blogs.windows.com/msedgedev/2021/08/10/compat2021-css-grid-gridng/).
{% endAside %}

## Allow new classes of applications to run on the web

[Project Fugu](/fugu-status/) is an effort that aims to allow new classes of applications to the web. 

> You can keep up to date with progress on the Fugu APIs at [fugu-tracker.web.app](https://fugu-tracker.web.app/)

Reviewer: ```suggestion
You can keep up to date with progress on the Fugu APIs at [fugu-tracker.web.app](https://fugu-tracker.web.app/).
```

Adobe recently announced that they are bringing Photoshop to the web, 
a feat made possible by collaboration between Adobe, Google, and web standards organizations. 

> Find out more in the post [Photoshop's Journey to the web](/ps-on-the-web/).

Reviewer: ```suggestion
Find out more in the post [Photoshop's journey to the web](/ps-on-the-web/).
```

## Ensure privacy

> It's difficult for developers to meet the growing need for privacy when many applications were developed using third-party cookies and solutions that weren't designed with privacy in mind.

Reviewer: ```suggestion
Many common web functions rely on third-party cookies and other cross-site tracking mechanisms that weren’t designed with privacy in mind, making it difficult for developers to meet the growing need for privacy.
```

Container queries look familiar to anyone who has ever used a media query to create a responsive design. 
However, inside of querying the viewport size, they allow you to query the size of the container your component is in. 
Google has been working on a trial implementation of the emerging specification, behind the `#enable-container-queries` flag in Chrome.

Read a blog post covering 
[RenderingNG in detail](https://developer.chrome.com/blog/renderingng/), 
or watch [this video](https://www.youtube.com/watch?v=sUbJPHYKZkU) 
to get all the key details of how this work unlocks implementation of features that were once thought impossible to land.

## Help you to create beautiful and responsive sites

The [new responsive design](/new-responsive/) is so much more than just screen—or container—size. 
You can learn how to create modern responsive sites in our course 
[Learn Design](/learn/design/). 
The first five modules launch today, the rest will be posted over the coming weeks.

New CSS properties have landed in Chrome and other browsers this year to make creating beautiful experiences easier. 

> Find out more about [accent-color](/accent-color/) and  [size-adjust](/css-size-adjust/).

Reviewer: ```suggestion
Find out more about [`accent-color`](/accent-color/) and [`size-adjust`](/css-size-adjust/).
```
